src/api/app/models/watchlist_items.rb

```diff
@@ -0,0 +1,9 @@
+class WatchlistItems
```

Comment:
Two more objects around `*ForWatchlistFinder`?

Reply:
What do you suggest?

Reply:
scope `watched_items` in `User`